setup.py

```diff
@@ -5,67 +5,10 @@
 version_sfl = SourceFileLoader("soundata.version", "soundata/version.py")
 version_module = version_sfl.load_module()
 
-with open("README.md", "r") as fdesc:
-    long_description = fdesc.read()
 
 if __name__ == "__main__":
     setup(
-        name="soundata",
         version=version_module.version,
-        description="Python library for loading and working with sound datasets.",
-        url="https://github.com/soundata/soundata",
         packages=find_packages(exclude=["test", "*.test", "*.test.*"]),
-        download_url="http://github.com/soundata/soundata/releases",
         package_data={"soundata": ["datasets/indexes/*"]},
```

Comment:
The idea is to host all the indexes remotely, so I think we'll have to remove this line, right? Although I think it's better if we leave this PR as is and change it in https://github.com/soundata/soundata/pull/169 directly. @genisplaja @hagenw

Reply:
Yes, I would tackle that in a separate pull request e.g. #169) to not mix too many things.